hw3/card.js

```diff
@@ -0,0 +1,32 @@
+function Card(name, newCardBalance){          
+    this.name = name;
+    isNaN(newCardBalance)?newCardBalance=0:true;
```

Comment:
better to understand when using regular condition statement instead of ternary operations

Reply:
Fixed.
Thanks, I  read this article - https://javascript.plainenglish.io/if-else-statements-vs-ternary-operators-in-javascript-d5238af4b516 , thus, I think I understood what my mistake was. As a rule, the ternary operator is used as a replacement for `if + else`. Otherwise, it is better to use an `if else statement`. In this case, I used the ternary operator is not advisable, since I add an extra check `else`, but I only need to execute with one` if.`
```suggestion
    if(isNaN(newCardBalance)){
        newCardBalance = 0;
    };
```